src/stavenote.ts

```diff
@@ -1089,6 +1096,10 @@ export class StaveNote extends StemmableNote {
     ctx.closeGroup();
   }
 
+  getStaveNoteScale(): number {
```

Comment:
Can you add a comment here that this is primarily used as the scaling factor for grace notes?

Reply:
Done